internal/users/localentries/localgroups.go

```diff
@@ -114,142 +140,167 @@ func existingLocalGroups(user, groupPath string) (groups []string, err error) {
 		if t == "" {
 			continue
 		}
-		elems := strings.Split(t, ":")
-		if len(elems) != 4 {
-			return nil, fmt.Errorf("malformed entry in group file (should have 4 separators): %q", t)
+		elems := strings.SplitN(t, ":", 4)
+		if len(elems) < 4 {
+			return nil, fmt.Errorf("malformed entry in group file (should have 4 separators, got %d): %q", len(elems), t)
```

Comment:
We should print a warning here (or just silently ignore the entry) instead of returning an error, else the ongoing request will fail if there is an invalid group entry in the group file.

The shadow-utils seem to silently ignore invalid lines: https://salsa.debian.org/debian/shadow/blob/4e64c42f1eb9a7c5f238e4b84965551c3976c144/lib/sgetpwent.c#L75-L85